_pages/contact.md

```diff
@@ -4,19 +4,64 @@ permalink: /contact/
 description: meta.contact_us.description
 ---
 
-<div>
-  <div class="container">
-    <h1>
-      {% t meta.contact_us.title %}
-    </h1>
-  </div>
-</div>
+{% capture heading %}
+{% t contact.hero.heading %}
+{% endcapture %}
```

Comment:
Can we indent inside the capture block?
```suggestion
{% capture heading %}
  {% t contact.hero.heading %}
{% endcapture %}
```

Reply:
> Can we indent inside the capture block?

👍  Updated in e516a0d .

I'm sensing a lint is in order 😉

Reply:
> I'm sensing a lint is in order 😉

Yes! Would love for this to be automatic! But it looks like [prettier won't support](https://github.com/prettier/prettier/issues/5680#issuecomment-647406669) and I wasn't able to find any plugins